v2/http-middleware/security.go

```diff
@@ -150,19 +158,11 @@ func getUserIDByToken(ctx context.Context, accessToken string) (_ string, err er
 		return
 	}
 
-	req, err := http.NewRequest(http.MethodGet, baseURL+endpoint, nil)
-	if err != nil {
-		err = errors.Wrap(err, "failed to create new HTTP request")
-		return
-	}
-
-	req = req.WithContext(ctx)
-	req.Header.Set("Content-Type", "application/json")
-	req.Header.Set("Authorization", accessToken)
-
-	client := &http.Client{Transport: &ochttp.Transport{}}
+	req := rest.Get(baseURL+endpoint).
```

Comment:
If you use the WithBaseURL as mentioned before maybe we can inline the endpoint constant here.